Makefile

```diff
@@ -1,107 +0,0 @@
-PROJECT ?= piraeus-operator
-REGISTRY ?= piraeusdatastore
-TAG ?= latest
-NOCACHE ?= false
-
-# Some operator-sdk versions struggle when these are not set, even if go
-# is perfectly content with using sane defaults. So we just query go on
-# what these defaults are
-GOROOT ?= $(shell go env GOROOT)
-GOPATH ?= $(shell go env GOPATH)
-OPERATORSDK ?= operator-sdk
-
-OPERATORSDKCMD := GOROOT=$(GOROOT) GOPATH=$(GOPATH) $(OPERATORSDK)
-
-help:
-	@echo "Useful targets: 'update', 'upload'"
-
-all: update upload
-
-.PHONY: update
-update:
-	$(OPERATORSDKCMD) build --image-build-args "--no-cache=$(NOCACHE)" $(PROJECT):$(TAG)
```

Comment:
Don't delete the whole makefile. Just replace this line with `docker build -t $(PROJECT):$(TAG) --no-cache=$(NOCACHE) -f build/Dockerfile.multiarch .`

Reply:
Needs to changed to `-f build/Dockerfile` now